addon/components/ember-popper-base.js

```diff
@@ -32,7 +35,7 @@ export default class EmberPopperBase extends Component {
    * The popper element needs to be moved higher in the DOM tree to avoid z-index issues.
    * See the block-comment in the template for more details.
    */
-  @property popperContainer = null
+  @property popperContainer = '.ember-application'
```

Comment:
I'd love a comment here just to explain to newbies that the root element always gets `'.ember-application'`, making this the highest element in the DOM tree we're able to attach ember-popper's to